test/integration/remote-config.spec.ts

```diff
@@ -287,4 +290,68 @@ describe('admin.remoteConfig', () => {
       expect(newTemplate.parameterGroups).to.deep.equal(VALID_PARAMETER_GROUPS);
     });
   });
+
+  describe('getServerTemplate', () => {
+    it('should return the most recently published template', () => {
+      // TODO: use global mock data once there's a REST API for server templates.
+      // Until then, copy/paste this into test server.
+      const VALID_TEMPLATE = {
+        'conditions': [
+          {
+            'name': 'ios',
+            'expression': "device.os == 'ios'"
+          },
+          {
+            'name': 'android',
+            'expression': "device.os == 'android'"
+          }
+        ],
+        'etag': 'etag-123-456',
+        'parameters': {
+          'holiday_promo_enabled': {
+            'defaultValue': {
+              'useInAppDefault': true
+            },
+            'description': 'promo indicator',
+            'valueType': 'STRING'
+          },
+          'welcome_message': {
+            'conditionalValues': {
+              'android': {
+                'value': 'welcome android text'
+              },
+              'ios': {
+                'value': 'welcome ios text'
+              }
+            },
+            'defaultValue': {
+              'value': 'welcome text 1706578886883'
+            },
+            'valueType': 'STRING'
+          }
+        },
+        'version': {
+          'description': 'asd'
+        }
+      };
+
+      const rc = getRemoteConfig();
+
+      // Set FIREBASE_REMOTE_CONFIG_URL_BASE env var to point this at a local server,
```

Comment:
Should this be in /** ... */ as users will need to set/change FIREBASE_REMOTE_CONFIG_URL_BASE as needed?

```suggestion
      /**
        * Set a `FIREBASE_REMOTE_CONFIG_URL_BASE` environment variable to direct
        * template evaluation to a local server, for example:
        * 
        *     `FIREBASE_REMOTE_CONFIG_URL_BASE=http://localhost:3000 npm run integration`
        * 
        * Note that this applies to all Remote Config endpoints, so if you're using a
        * server that doesn't implement all endpoints, filter for this specific test.
        */
```

Reply:
TLDR: yes.

I was thinking multi-line comments are mostly useful for JSDoc formatting, but I see the style guide recommends "Use /** JSDoc */ comments for documentation, i.e. comments a user of the code should read." I was intending for this to be documentation, albeit for obscure functionality, so JSDoc comments are appropriate.